Dshop/apps/products_catalogue/views.py

```diff
@@ -1,3 +1,15 @@
-#from django.shortcuts import render
+from django.shortcuts import render
+from django.views import View
+from .models import Product
 
 # Create your views here.
+class ProductListView(View):
+    template_name = 'products_catalogue/products_list.html'
+    model = Product
+    def get(self, request):
+        all_products = Product.objects.all()
```

Comment:
Products have "is_active = models.BooleanField(default=True)" field, which is ignored here. 
Please fix.

Reply:
done! :)